core/data/tests/can_generate_algebraic_enum/output.go

```diff
@@ -121,11 +121,11 @@ func NewAdvancedColorsTypeVariantReallyCoolType(content *ItemDetailsFieldValue)
 type AdvancedColors2Types string
 const (
 	// This is a case comment
-	AdvancedColors2TypeVariantString AdvancedColors2Types = "String"
-	AdvancedColors2TypeVariantNumber AdvancedColors2Types = "Number"
-	AdvancedColors2TypeVariantNumberArray AdvancedColors2Types = "NumberArray"
+	AdvancedColors2TypeVariantString AdvancedColors2Types = "string"
```

Comment:
Can you add a test for title case as well, just for completeness sake. I see everything was TitleCase before this change, so it makes sense to check that is still behaviour we can achieve somehow, especially since we might need to consider backwards compatibility for internal tools using this typeshare feature.

Reply:
You mean a test case in which no serde renaming is used or a test in which renaming is title case? Or both?

Reply:
Where no renaming is used, as the most important.

Reply:
There's already a case in that test itself. Check `AdvancedColors` [here](https://github.com/1Password/typeshare/pull/233/files#diff-ec3820b150bb49f5f3b81093a7cf9d95954e5a433c2329dfb6203704fe8c73d7R9-R18). Notice how it wasn't changed as part of this PR.